File: module/koin/src/main/java/com/tory/module/hilt/model/GankApiResult.java
package com.tory.demo.jetpack.model;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class GankApiResult{

	@SerializedName("error")
	private boolean error;

	@SerializedName("results")
	private List<GankItem> results;

	public void setError(boolean error){
		this.error = error;
	}

	public boolean isError(){
		return error;
	}

	public void setResults(List<GankItem> results){
		this.results = results;
	}

	public List<GankItem> getResults(){
		return results;
	}

	@Override
 	public String toString(){
		return 
			"GankApiResult{" + 
			"error = '" + error + '\'' + 
			",results = '" + results + '\'' + 
			"}";
		}
}

File: demo/jetpackdemo/src/main/java/com/tory/demo/jetpack/model/GankApiResult.kt
package com.tory.demo.jetpack.model

import com.google.gson.annotations.SerializedName

data class GankApiResult(
    @SerializedName("error")
    var isError: Boolean = false,
    @SerializedName("results")
    var results: List<GankItem>? = null
)
